Atualizar um Participante associado a um Plano de Trabalho gera erro de dependência de chaves no BD

Passos para reproduzir:
1. criar um Participante;
2. criar um Plano de Trabalho a ele associado;
3. atualizar o Participante.

Erro apresentado:
```
FAILED participantes_test.py::test_update_participante_with_existing_pt - AssertionError: Dependency rule on column 'participante.origem_unidade' tried to blank-out primary key column 'plano_trabalho.origem_unidade' on instance '<PlanoTrabalho at 0x...>'
```

--- a/src/crud.py
+++ b/src/crud.py
@@ -498,14 +498,9 @@ async def update_participante(
             .filter_by(matricula_siape=participante.matricula_siape)
         )
         db_participante = result.unique().scalar_one()
-        data_insercao = db_participante.data_insercao
-        await session.delete(db_participante)
-        await session.commit()
-        # create a new
-        db_participante = models.Participante(**participante.model_dump())
-        db_participante.data_insercao = data_insercao
+        for field, value in participante.model_dump().items():
+            setattr(db_participante, field, value)
         db_participante.data_atualizacao = datetime.now()
-        session.add(db_participante)
         await session.commit()
         await session.refresh(db_participante)
     return schemas.ParticipanteSchema.model_validate(db_participante)

--- a/tests/participantes_test.py
+++ b/tests/participantes_test.py
@@ -234,6 +234,28 @@ def test_put_duplicate_participante(
     assert_equal_participante(response.json(), input_part)
 
 
+def test_update_participante_with_existing_pt(
+    truncate_participantes,  # pylint: disable=unused-argument
+    example_part,  # pylint: disable=unused-argument
+    example_pt,  # pylint: disable=unused-argument
+    input_part: dict,
+    header_usr_1: dict,
+    client: Client,
+):
+    """Atualiza um participante existente, sendo que o participante já
+    possui um Plano de Trabalho a ele associado.
+    """
+    response = client.put(
+        f"/organizacao/{input_part['origem_unidade']}"
+        f"/{input_part['cod_unidade_autorizadora']}"
+        f"/{input_part['cod_unidade_lotacao']}"
+        f"/participante/{input_part['matricula_siape']}",
+        json=input_part,
+        headers=header_usr_1,
+    )
+    assert response.status_code == status.HTTP_200_OK
+
+
 def test_create_participante_inconsistent(
     truncate_participantes,  # pylint: disable=unused-argument
     input_part: dict,

